src/interfaces/earnings.ts

```diff
@@ -30,6 +30,9 @@ import {
 const BigZero = new BigNumber(0);
 
 export class EarningsInterface<C extends ChainId> extends ServiceInterface<C> {
+  /*
+   * Deperacated function should removed in the future
```

Comment:
Can we use the deprecated tag for this? https://jsdoc.app/tags-deprecated.html

Reply:
here, I fixed it in this `pr`
[https://github.com/yearn/yearn-sdk/pull/238](prl)

Reply:
Thanks, you can just commit to the same PR branch in future so you don't need to make more PRs